main/main.ino

```diff
@@ -984,6 +987,10 @@ void setup_wifi() {
 #  if defined(ESP32) && defined(ZgatewayBT)
     if (failure_number_ntwk > maxConnectionRetryWifi && lowpowermode)
       lowPowerESP32();
+#  else
```

Comment:
Rather than an #else maybe we should just condition this function execution to not being in lowpowermode.
This would correct the result of 3) from my comment

Reply:
Let me recreate this and submit an update.  Tks for taking a moment to review

Reply:
Maybe we could have also a different macro for the max connection retry than `maxConnectionRetryWifi`, with a bigger number like 30 to avoid repetitive restart

Reply:
Something like a maximumRetryWatchdog, and set it to be 2 x maxConnectionRetryWifi ?

Reply:
2 x + 1

Reply:
yes would be cool